src/solver/conv_hip_implicit_gemm_grouped_bwd_xdlops.cpp

```diff
@@ -460,7 +460,7 @@ bool ConvHipImplicitGemmGroupBwdXdlops::IsApplicable(
     [[maybe_unused]] const ProblemDescription& problem) const
 {
 #if MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
-    if(miopen::IsDisabled(ENV(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_GROUP_BWD_XDLOPS)))
+    if(miopen::IsEnabled(ENV(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_GROUP_BWD_XDLOPS)))
```

Comment:
@msaudulhassan interesting way to disable a solver ;-P I like it that we can test the fix later by enabling that env var explicitly.

Reply:
@junliume PS: I'll set this PR to Open once I have tested it on the detectron2 workload